crates/atiny-tests/suite/bodyless.expect

```diff
@@ -1,7 +1,9 @@
 
 [error]: type mismatch between '()' and 'Int'
+
     ┌─> bodyless.at:13:16
     │
  13 │fn main4 : Int {
```

Comment:
I think that you could just add a space between the "|" and the line because it looks like a little bit ugly.